netbox/core/tables/plugins.py

```diff
@@ -1,13 +1,17 @@
+from datetime import datetime
 import django_tables2 as tables
+from django.contrib.humanize.templatetags.humanize import naturalday
 from django.utils.translation import gettext_lazy as _
 from netbox.tables import BaseTable
 
 __all__ = (
-    'PluginTable',
+    'CertifiedPluginTable',
+    'InstalledPluginTable',
+    'PluginVersionTable',
 )
 
 
-class PluginTable(BaseTable):
+class InstalledPluginTable(BaseTable):
```

Comment:
IMO we can ditch the plugins list on the system view now that we have a dedicated view for them.